test/integration/integration_admin_test.cc

```diff
@@ -136,13 +147,13 @@ TEST_P(IntegrationAdminTest, Admin) {
               testing::HasSubstr("# TYPE envoy_cluster_upstream_cx_active gauge\n"));
   EXPECT_THAT(
       response->body(),
-      testing::HasSubstr("envoy_cluster_upstream_cx_active{envoy_cluster_name=\"cds\"} 0\n"));
+      testing::HasSubstr("envoy_cluster_upstream_cx_active{envoy_cluster_name=\"cluster_0\"} 0\n"));
   response = IntegrationUtil::makeSingleRequest(lookupPort("admin"), "GET", "/clusters", "",
                                                 downstreamProtocol(), version_);
   EXPECT_TRUE(response->complete());
   EXPECT_STREQ("200", response->headers().Status()->value().c_str());
   EXPECT_THAT(response->body(), testing::HasSubstr("added_via_api"));
-  EXPECT_THAT(response->body(), testing::HasSubstr("version_info::\n"));
+  EXPECT_THAT(response->body(), testing::HasSubstr("version_info::static\n"));
```

Comment:
This is the one functional change I'm unsure about.  Hadn't tracked down the cause.

Reply:
I'm guessing it's because server.json has CDS setup but there is no version info yet. This raises a point that for better or worse server.json has a bunch of random stuff in it mainly just to make sure that basic config elements don't break even if there is no usable upstream CDS server, stats backend, etc. We should try to make sure that we don't lose this "coverage."

Reply:
Sorry, I'm not sure I'm understanding the ask here.  
Are you asking that the utility class preserve standard things like health checking + admin config from server.json rather than the relevant tests adding them, or something else?

Reply:
I'm not really asking for anything. I was just making a comment that there is some test coverage that we had gained from the server.json file that we may lose with the simplistic basic config template and targeted additions we are now using for v2 tests (as seen by this delta). It would be worth an audit of the delta to see if we are losing anything and want to bring any of it back in a targeted "sanity test" of random features. We can do this in follow ups...

Reply:
Ahhh, so we're missing coverage of config loading if those features aren't explicitly tested.  Sure I'll take a look at what's missing once I get the rest of the tests moved over.